README.md

```diff
@@ -372,14 +390,15 @@ void GREENTEA_SETUP(const int timeout, const char *host_test_name) {
 ```
 * Corresponding log:
 ```
-[1458565465.35][SERI][INF] reset device using 'default' plugin...
-[1458565465.60][SERI][INF] wait for it...
-[1458565466.60][CONN][INF] sending preamble '2f554b1c-bbbf-4b1b-b1f0-f45493282f2c'
-[1458565466.60][SERI][TXD] mbedmbedmbedmbedmbedmbedmbedmbedmbedmbed
-[1458565466.60][SERI][TXD] {{__sync;2f554b1c-bbbf-4b1b-b1f0-f45493282f2c}}
-[1458565466.74][CONN][INF] found SYNC in stream: {{__sync;2f554b1c-bbbf-4b1b-b1f0-f45493282f2c}}, queued...
-[1458565466.74][HTST][INF] sync KV found, uuid=2f554b1c-bbbf-4b1b-b1f0-f45493282f2c, timestamp=1458565466.743000
-[1458565466.74][CONN][RXD] {{__sync;2f554b1c-bbbf-4b1b-b1f0-f45493282f2c}}
+[1530040532.96][GLRM][INF] remote resources reset...
+[1530040535.26][GLRM][TXD] mbedmbedmbedmbedmbedmbedmbedmbedmbedmbed
+[1530040535.26][CONN][INF] sending up to 5 __sync packets (specified with --sync=5)
+[1530040535.26][CONN][INF] sending preamble 'f1ee68c1-86f4-4025-a5af-e755e03802c6'
+[1530040535.26][GLRM][TXD] {{__sync;f1ee68c1-86f4-4025-a5af-e755e03802c6}}
+[1530040536.32][CONN][RXD] Host Key parsed
```

Comment:
If we decide to drop this section (as I commented here: https://github.com/ARMmbed/mbed-os/pull/7340/files#r241550112), then this comment is just a reminder to remove this from the docs as well.